libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -142,6 +144,12 @@ namespace eosio { namespace testing {
       };
    }
 
+   struct produce_block_result_t {
+      signed_block_ptr                   block;
+      transaction_trace_ptr              onblock_trace;
```

Comment:
I see no reason to have this separate, should just be the first trace in `traces`.

Reply:
Sure, done.

Reply:
Unfortunately, this causes a failure in `forked_tests.cpp` because the old API returned a vector of traces *not* including the `onblock` trace.
Should I re-separate the `onblock` trace, or modify `forked_tests.cpp` to use the new format.

Reply:
Either one works, we often handle `onblock` differently, so having in its own variable is fine.

Reply:
Thanks. Updating it to use its own variable.